### PR TITLE
Create README.ko.md

### PR DESCRIPTION
**Pull request by Minhyeok-Oh**
Hi. Official I am a college student in Korea. I've translated the README.md file to make it easier for Korean contributors to access and contribute to the docsify.
I'm still a beginner, so I have a lot. Please let me know if there is anything you need to fix. If you wish, we will translate other documents.

**Summary**

**What kind of change does this PR introduce?** (check at least one)

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update
- [ ] Refactor
- [O] Docs
- [ ] Build-related changes
- [ ] Other, please describe:

If changing the UI of default theme, please provide the **before/after** screenshot:

**Does this PR introduce a breaking change?** (check one)

- [ ] Yes
- [O] No

If yes, please describe the impact and migration path for existing applications:

**The PR fulfills these requirements:**

- [O] When resolving a specific issue, it's referenced in the PR's title (e.g. `fix #xxx[,#xxx]`, where "xxx" is the issue number)

You have tested in the following browsers: (Providing a detailed version will be better.)

- [ ] Chrome
- [ ] Firefox
- [ ] Safari
- [ ] Edge
- [ ] IE

If adding a **new feature**, the PR's description includes:

- [ ] A convincing reason for adding this feature
- [ ] Related documents have been updated
- [ ] Related tests have been updated

To avoid wasting your time, it's best to open a **feature request issue** first and wait for approval before working on it.

**Other information:**

---

* [O] DO NOT include files inside `lib` directory.

### CODE REVIEW

File: README.ko.md [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+<p align="center">
+  <a href="https://opencollective.com/docsify/order/3254">
+    <img src="https://opencollective.com/docsify/tiers/gold-sponsor.svg?avatarHeight=36">
+  </a>
+</p>
+
+## 링크
+
+- [`develop` branch preview](https://docsifyjs.netlify.com/)
+- [Documentation](https://docsify.js.org)
```
Comment (anikethsaha) [inline README.ko.md]: Can you translate the word `Documentation` also ?

File: README.ko.md [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+이 프로젝트는 문서를 사용하여 사이트를 생성합니다. pull request를 환영합니다
+
+옆 링크를 클릭하세요. [awesome-docsify](https://github.com/docsifyjs/awesome-docsify#showcase)
+
+## 유사한 프로젝트
+
+| Project                                          | Description                              |
+| ------------------------------------------------ | ---------------------------------------- |
+| [docute](https://github.com/egoist/docute)       | 📜 Effortlessly documentation done right |
+| [docpress](https://github.com/docpress/docpress) | Documentation website generator          |
```
Comment (anikethsaha) [inline README.ko.md]: Can you translate the descriptions too

File: README.ko.md [hunk 3]
```diff
@@ -0,0 +1,111 @@
...
+</p>
+
+## 링크
+
+- [`develop` branch preview](https://docsifyjs.netlify.com/)
+- [Documentation](https://docsify.js.org)
+- [CLI](https://github.com/docsifyjs/docsify-cli)
+- CDN: [UNPKG](https://unpkg.com/docsify/) | [jsDelivr](https://cdn.jsdelivr.net/npm/docsify/) | [cdnjs](https://cdnjs.com/libraries/docsify)
+- [Awesome docsify](https://github.com/docsifyjs/awesome-docsify)
+- [Community chat](https://gitter.im/docsifyjs/Lobby)
```
Comment (anikethsaha) [inline README.ko.md]: try to translate `CLI`, `CDN`, `Awesome docsify`, and `Community chat` too

File: README.ko.md (hunk 1 above)
Comment (Minhyeok-Oh) [reply README.ko.md]: complete.

File: README.ko.md (hunk 2 above)
Comment (Minhyeok-Oh) [reply README.ko.md]: complete.

File: README.ko.md (hunk 3 above)
Comment (Minhyeok-Oh) [reply README.ko.md]: complete

**Comment by anikethsaha** [comment]
> it's better if you can translate the docs page of the website. Translating the readme might make the repo a bit messy.
> 
> We do have translations on the website. Better keep it there itself.

@Minhyeok-Oh lets leave the readme for now. Go for the site docs.

**Comment by anikethsaha** [comment]
https://github.com/docsifyjs/docsify/pull/955#issuecomment-557787563

**Pull request closed without merging**